plugin/session_buffer.py

```diff
@@ -428,14 +428,20 @@ def _do_document_link_async(self, view: sublime.View, version: int) -> None:
 
     def _on_document_link_async(self, view: sublime.View, response: list[DocumentLink] | None) -> None:
         self._document_links = response or []
+        self.redraw_document_links_async()
+
+    def redraw_document_links_async(self) -> None:
         if self._document_links and userprefs().link_highlight_style == "underline":
-            view.add_regions(
-                "lsp_document_link",
-                [range_to_region(link["range"], view) for link in self._document_links],
-                scope="markup.underline.link.lsp",
-                flags=DOCUMENT_LINK_FLAGS)
+            view = self.some_view()
+            if not view:
+                return
+            regions = [range_to_region(link["range"], view) for link in self._document_links]
+            for sv in self.session_views:
+                sv.view.add_regions(
+                    "lsp_document_link", regions, scope="markup.underline.link.lsp", flags=DOCUMENT_LINK_FLAGS)
```

Comment:
Nice fix to region rendering. Ideally would be its own PR to keep records clear but no pressure.

Reply:
On the other hand if we keep it out of the commit history, maybe we can keep it a secret that this bug existed ;)